src/D3MHub.sol

```diff
@@ -600,4 +466,50 @@ contract D3MHub {
         ilks[ilk].culled = 0;
         emit Uncull(ilk, wad);
     }
+
```

Comment:
[red team comment] we were thinking we might want to add a `require(end.debt() == 0, "end-debt-set")` to the requires in `uncull`. We know it will not matter because we can't call `exec` once debt > 0, but this would prevent the vat balance changes and reduce the amount of context required to understand `uncull`.  Since it is not a hot path the gas cost shouldn't matter.

Reply:
As we discussed, we shouldn't add this restriction as we can still `hub.uncull(ilk)`, `end.cage(ilk)`. `end.skim(ilk)` and make this collateral available for DAI holders redemption even after `end.wait` passes.